src/FileService.Broker/Consumers/AddImageConsumer.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.FileService.Business.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.RequestMappers.Interfaces;
+using LT.DigitalOffice.Kernel.Broker;
 using MassTransit;
-using System;
+using Microsoft.AspNetCore.Http;
 using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.FileService.Broker.Consumers
 {
     public class AddImageConsumer : IConsumer<IAddImageRequest>
     {
-        public Task Consume(ConsumeContext<IAddImageRequest> context)
+        private readonly IAddNewImageCommand _command;
+        private readonly IImageRequestMapper _mapper;
+
+        private object GetImageId(IAddImageRequest request)
+        {
+            var imageRequest = _mapper.Map(request);
+
+            var imageId = _command.Execute(imageRequest, request.UserId);
+
+            return new
+            {
+                Id = imageId
+            };
+        }
+
+        public AddImageConsumer(
+            IAddNewImageCommand command,
+            IImageRequestMapper mapper)
         {
-            throw new NotImplementedException();
+            _command = command;
+            _mapper = mapper;
+        }
```

Comment:
Constructor between methods?

Reply:
private methods at the beginning of the class and public at the end